BUILDING.md

```diff
@@ -1,4 +1,91 @@
-# Building Habitat from source
+# Setting up the environment
+
+## Officially supported environment: Ubuntu Latest (17.10/Artful)
+
+([Other environments](#Unsupported-environments) work, but are not officially supported)
+
+You can run this environment natively or in VM. [ISOs for both server and desktop versions are available here](http://releases.ubuntu.com/17.10/).
+This installation method uses as many packages from Ubuntu as possible.
+If you don't have it, you'll first need to install `git`:
+```
+sudo -E apt-get install -y --no-install-recommends git
+```
```

Comment:
maybe just link to https://git-scm.com/book/en/v2/Getting-Started-Installing-Git ?